tests/metrics/README.md

```diff
@@ -4,19 +4,21 @@
 Some tests require you to enable the debug mode of cc-oci-runtime. To enable it, please take a look at:
 https://github.com/01org/cc-oci-runtime#18running-under-docker
 
-To run the metrics, just run:
+
+### Run all metrics tests
+To run the metrics, please use Makefile rule `metrics-tests` on the top level of this repository:
 
 ```bash
-$ cd tests/metrics
-$ ./run_docker_metrics
+$ sudo -E PATH=$PATH make metrics-tests
 ```
 
 This will run all metrics tests and generete a `results` directory.
 
-Each file in the `result` directory contains the result for a test and has the format of a CSV.
+Each file in the `results` directory contains the result for a test and has the format of a CSV.
 At the end of each file you will find the result average of all the data collected by the test.
 
-You can also run each tests script separately. e.g.
+You will be able to run individual tests after `make metrics-tests` has been exectued (so necessary
```

Comment:
Execute `make metrics-tests` to generate the necessary files needed to run individual tests. For example: